sdk/personalizer/Azure.AI.Personalizer/api/Azure.AI.Personalizer.netstandard2.0.cs

```diff
@@ -1,5 +1,73 @@
 namespace Azure.AI.Personalizer
 {
+    public partial class DecisionContext
+    {
+        public DecisionContext() { }
+        public DecisionContext(Azure.AI.Personalizer.PersonalizerRankMultiSlotOptions rankRequest, System.Collections.Generic.Dictionary<string, System.Collections.Generic.IList<object>> slotIdToFeatures) { }
+        public DecisionContext(System.Collections.Generic.IEnumerable<object> contextFeatures, System.Collections.Generic.List<Azure.AI.Personalizer.PersonalizerRankableAction> rankableActions) { }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("FromUrl")]
+        public System.Collections.Generic.List<string> ContextFeatures { get { throw null; } }
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("_multi")]
+        public Azure.AI.Personalizer.DecisionContextDocument[] Documents { get { throw null; } set { } }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("_slots")]
+        public Azure.AI.Personalizer.DecisionContextDocument[] Slots { get { throw null; } set { } }
+    }
+    public partial class DecisionContextDocument
+    {
+        public DecisionContextDocument(string id, System.Collections.Generic.List<string> json, string slotId, System.Collections.Generic.List<string> slotJson) { }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("f")]
+        public System.Collections.Generic.Dictionary<string, float[]> FloatFeatures { get { throw null; } }
```

Comment:
Regarding `FloatFeatures`, it would be better if the property name did not indicate the datatype it is working with.

Reply:
Changing the name to Features.